### DISCUSSION TITLE
Week 12

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#236] [General]
I'm feeling pretty satisfied with how my final page is looking. The last little stretch of necessary progress to make my page look really good began by meeting with Professor Synder. This was a huge help and I feel like every time I've met with a Professor during this course it has greatly benefitted me. By getting direction on tasks I've been hung up on, usually being guided through processes that would've taken me hours figuring them out on my own, only taking a few minutes with help from them. 

I was able to sync changes, adjust alignment, and learn why certain CSS selectors I had been using weren't working. Now it's really just making final decisions on stylizing things which I could unfortunately change and adjust forever haha. 

I decided I am going to be publishing the site once I've finished, when I do that I'll provide a link for everyone to be able to see the work I've done.

**Comment by beto0890** [comment]
I look forward to getting you the link and seeing your final product. I am sure is looks great!

**Comment by gabebott** [comment]
I would love to take a look at it. If you end up adding a link let me know! Everything sounds great! I look forward to seeing it.
